pdns/lua-record.cc

```diff
@@ -1245,6 +1245,7 @@ static void setupLuaRecords(LuaContext& lua) // NOLINT(readability-function-cogn
   lua.writeFunction("geoiplookup", [](const string &ip, const GeoIPInterface::GeoIPQueryAttribute attr) {
     return getGeo(ip, attr);
   });
+  lua.writeVariable("GeoIPQueryAttribute", std::unordered_map<std::string, int>{{"ASn", GeoIPInterface::GeoIPQueryAttribute::ASn}, {"City", GeoIPInterface::GeoIPQueryAttribute::City}, {"Continent", GeoIPInterface::GeoIPQueryAttribute::Continent}, {"Country", GeoIPInterface::GeoIPQueryAttribute::Country}, {"Country2", GeoIPInterface::GeoIPQueryAttribute::Country2}, {"Name", GeoIPInterface::GeoIPQueryAttribute::Name}, {"Region", GeoIPInterface::GeoIPQueryAttribute::Region}, {"Location", GeoIPInterface::GeoIPQueryAttribute::Location}});
```

Comment:
This might be cleaner as 8-10 lines. Let's see what the formatter says.

Reply:
It seems that the formatter doesn't complain about that. I can manually format it if you insist.
By the way, `dnsdist-lua-vars.cc` also contains code in this style.
https://github.com/PowerDNS/pdns/blob/2c5c5b828fcf6a50899c7100b86d7be20235f173/pdns/dnsdistdist/dnsdist-lua-vars.cc#L30-L42

Reply:
> By the way, `dnsdist-lua-vars.cc` also contains code in this style.

I just noticed the same. No change needed, thanks!